roles/confluent.ksql/tasks/main.yml

```diff
@@ -250,15 +255,22 @@
     group: "{{ksql_group}}"
     state: directory
     mode: 0640
+  when: not unprivileged_install|bool
```

Comment:
This `unprivileged_install` var needs to get into confluent.variables/defaults/main.yml w annotations- see https://github.com/confluentinc/cp-ansible/blob/6.2.x/DEVELOPMENT_GUIDE.md#variables

Reply:
someone recently recommended to me not to use double negatives as they are harder to read, i'm wondering if this should be;
```
when: privileged_install|bool
```
And privileged_install would obviously default to true

Also I see we also have 
```
when: systemd_mode == 'user'
```
So that var could be built out of the privileged_install var ie
```
systemd_mode: "{{ 'system' if privileged_install else 'user'}}"
```

Reply:
I agree on the principle and basically I'll do what you prefer. 
The many many `omit if unprivileged_install` are going to turn into `omit if not privileged_install else 'bla'`.

Reply:
ya `omit if not privileged` makes sense to me logically. It reads the same as `omit if unprivileged`. its the do something when not unprivileged that makes me have to think

Reply:
got it, no problem it's just a Replace All away